app-emulation/acbuild/acbuild-9999.ebuild

```diff
@@ -0,0 +1,37 @@
+# Copyright (c) 2015 CoreOS, Inc.
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=5
+CROS_WORKON_PROJECT="appc/acbuild"
+CROS_WORKON_REPO="git://github.com"
+CROS_WORKON_LOCALNAME="appc-acbuild"
+inherit cros-workon
+
+if [[ "${PV}" == 9999 ]]; then
+    KEYWORDS="~amd64 ~arm64"
+else
+    CROS_WORKON_COMMIT="e49e7de27443374a5356b7ccd9d9d7691fb5089f" # v0.2.2
+    KEYWORDS="amd64 arm64"
+fi
+
+DESCRIPTION="A build tool for ACIs"
+HOMEPAGE="https://github.com/appc/acbuild"
+SRC_URI=""
+
+LICENSE="Apache-2.0"
+SLOT="0"
+IUSE=""
+
+DEPEND=">=dev-lang/go-1.4.3"
+RDEPEND="sys-apps/kmod
+		app-crypt/gnupg
+		sys-apps/systemd"
+
+src_compile(){
+		./build
```

Comment:
Indentation should be a single tab.

Is there anything special about this build? Can it use our go eclass?